frontend-opal/components/PortfolioContainer/Instruments/Instruments.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { SafeAreaView, View, Text, TouchableOpacity, Image, FlatList } from 'react-native';
+import { Card } from 'native-base';
+import styles from './styles';
+
+function InstrumentsItem({ company, stockExchange,profileImg, positions, marketValue, unrlzedPLPercentage, unrlzedPL }){
```

Comment:
@WenyuD an InstrumentList component gets the instrumentList as prop which is cool, understandable
this should be the logic of the InstrumentItem component too, you can destructure that in the next line